.gitignore

```diff
@@ -1,2 +1,6 @@
-/dist/
-/node_modules/
+node_modules
+dist
+coverage
+
+*.log*
+.DS_Store
```

Comment:
You could add `.DS_Store` to your global ignore file. See here: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
I think there should be a newline at the end of this file.

Reply:
I'd rather include the `.DS_Store` to make sure anyone else who submits a PR doesn't accidentally commit it in the code.

Reply:
> I'd rather include the .DS_Store to make sure anyone else who submits a PR doesn't accidentally commit it in the code.

Make sense.